### PR TITLE
docker/docker-compose.yaml : Add fix for mongodb container on WSL

### PR DESCRIPTION
**Pull request by JenySadadia**
Added docker volume for mongodb.
Signed-off-by: Jeny Sadadia <jeny.sadadia@gmail.com>

### CODE REVIEW

**Comment by gctucker** [comment]
Tip: If you add something like "Fixes #43" in the PR description it should automatically link it with the issue.

**Comment by JenySadadia** [comment]
> Tip: If you add something like "Fixes #43" in the PR description it should automatically link it with the issue.

Thanks for the tip. I will use that from now onwards :)

**Comment by gctucker** [comment]
I'm hitting this error:
```
Successfully built dd4ee458c5d7
Successfully tagged docker_api:latest
Creating kernelci-db      ... error
Creating kernelci-redis ... 
Creating kernelci-api   ... 
Creating kernelci-storage ... 
Creating kernelci-redis   ... done
Creating kernelci-api     ... done
Creating kernelci-storage ... done

ERROR: for db1  Cannot create container for service db1: Conflict. The container name "/kernelci-db" is already in use by container "4d3cd3ea6adb23a9efb5be0cc7d189857936edfc8e5b879608fc0c8b5d79e409". You have to remove (or rename) that container to be able to reuse that name.
ERROR: Encountered errors while bringing up the project.
```
Maybe it's just a problem with my local setup, I'll try again from a clean setup but still it would be interesting to understand why this is happening.

**Comment by gctucker** [comment]
OK I just had to remove an old stale container.  For some reason it wasn't a problem with the volume using a bind-mounted directory.

File: docker/docker-compose.yaml [hunk 1]
```diff
@@ -17,11 +17,11 @@ services:
     env_file:
       - ../.env
 
-  db:
+  db1:
```
Comment (gctucker) [inline docker/docker-compose.yaml:20]: Why was this renamed to `db1`?  I think it should be OK to leave it as `db`.

**Comment by gctucker** [comment]
I renamed the service back to `db` and got this to work fine locally.  Of course this created a new empty Mongo database but that's normal since it's a different Docker volume (it's possible to transfer the data if needed, for example using mongodump / mongorestore).

File: docker/docker-compose.yaml (hunk 1 above)
Comment (JenySadadia) [reply docker/docker-compose.yaml]: I hit the same issue of ``` ERROR: for db  Cannot create container for service db: Conflict. The container name "/kernelci-db" is already in use ```. So I tried to rename the service to db1 and check.
It was for testing only.
Yes, we don't need to rename the service name.

**Comment by JenySadadia** [comment]
> I renamed the service back to `db` and got this to work fine locally. Of course this created a new empty Mongo database but that's normal since it's a different Docker volume (it's possible to transfer the data if needed, for example using mongodump / mongorestore).

Okay. I pushed the changes with 'db' service name.

**Review by gctucker — APPROVED** [review]
Tested OK locally using kernelci-pipeline.
